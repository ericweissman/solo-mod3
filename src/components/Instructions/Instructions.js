import React from 'react'

const Instructions = () => {
  return (
    <div className='instructions'>
      <h3>Welcome to AARPlur</h3>
      <p>Your one stop shop to learn about all the music that the young whippersnappers are boogying to!</p>
      <h5>Instructions</h5>
      <ul>
        <li>Select one of the music genres using the left-side menu</li>
        <li>If you click on the image on the card, it will play you a popular song from the artist. Make sure you have your volume turned on your computer!</li>
        <li>If you still want to learn more about the artist, click on the More Info button to visit their online encyclopedia page.</li>
      </ul>
      <p>We hope you enjoy learning about this newfangled robot music!</p>
    </div>
  )
}

export default Instructions