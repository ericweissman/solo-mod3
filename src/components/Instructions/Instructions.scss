.instructions {
  padding-top: 100px;
  display: flex;
  flex-direction: column;
  align-items: center;
  font-family: $secondary-font;
  color: $color-accent-light;
  text-align: center;
  p {
    padding: 10px;
    font-size: 20px;
    color: $text-white;
  }
  .instructions-area {
    display: flex;
    div {
      @extend .card;
      width: 200px;
      margin: 10px;
      background-color: $color-main;
      height: fit-content;
      p {
        padding: 0 10px 15px 10px;
        color: $text-white;
        font-size: 16px;
      }
    }

    h5 {
      text-decoration: underline;
      font-size: 22px;
      padding: 10px;
    }
  }

  h3 {
    font-family: $title-font;
    font-size: 35px;
    color: $color-accent;
    padding: 15px;
  }
}