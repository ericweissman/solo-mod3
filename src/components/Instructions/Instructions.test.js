import React from 'react'
import {shallow} from 'enzyme'
import Instructions from '../Instructions/Instructions'

describe('Instructions', () => {
  let wrapper

  it('should match the correct snapshot', () => {
    wrapper = shallow(<Instructions />)
    expect(wrapper).toMatchSnapshot()
  })
})