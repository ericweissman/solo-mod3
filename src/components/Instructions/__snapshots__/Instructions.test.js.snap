// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instructions should match the correct snapshot 1`] = `
<div
  className="instructions"
>
  <h3>
    Welcome to AARPlur
  </h3>
  <p>
    Your one stop shop to learn about all the music that the young whippersnappers are boogying to!
  </p>
  <div
    className="instructions-area"
  >
    <div>
      <h5>
        Selecting Genres
      </h5>
      <p>
        Select one of the music genres using the left-side menu
      </p>
    </div>
    <div>
      <h5>
        Listening to Music
      </h5>
      <p>
        If you click on the image on the card, it will play you a popular song from the artist. Make sure you have your volume turned on your computer!
      </p>
    </div>
    <div>
      <h5>
        Getting More Info
      </h5>
      <p>
        Click on the More Info button to visit their online encyclopedia page
      </p>
    </div>
    <div>
      <h5>
        Adding Favorites
      </h5>
      <p>
        Click on the Add Favorites button on a card to save it to your Favorites! You can see your saved tracks when you click on Favorites.
      </p>
    </div>
  </div>
</div>
`;
